dynamics/src/include/MEBParameters.hpp

```diff
@@ -51,21 +40,9 @@ class MEBParameters {
 
     MEBParameters()
     {
-        rho_ice = 900.0; //!< Sea ice density
-        rho_atm = 1.3; //!< Air density
-        rho_ocean = 1026.0; //!< Ocean density
-
-        C_atm = 1.2e-3; //!< Air drag coefficient
-        C_ocean = 5.5e-3; //!< Ocean drag coefficient
         ocean_turning_angle = 25.; //!< Ocean turning angle
-        ocean_turning_angle = 0.0;
+        ocean_turning_angle = 0.0; // FIXME decide between ocean turning angles
```

Comment:
The turning angle belongs in the base class

Reply:
Done.